src/passes.jl

```diff
@@ -306,6 +306,7 @@ end
 function process!(::SetupBuildDirectory, page::Page, content, block)
     dir = dirname(page.build)
     isdir(dir) || mkpath(dir)
+    addstaticfiles!(dir)
```

Comment:
Any thoughts on:

> append! staticfiles to document.pages

I think this would be my preferred refactor, if it were possible.